src/pages/word-duel.jsx

```diff
@@ -14,11 +14,16 @@ const EvaluationColor = {
 export default function WordDuel() {
     // const [ word, setWord ] = useState(generateNewWord())
     const word = "butts";
+    const [active, setActive] = useState("".padEnd(word.length));
     const [guesses, setGuesses] = useState([]);
     const [evaluations, setEvaluations] = useState([]);
     const [gameOver, setGameOver] = useState(false);
     const [message, setMessage] = useState("");
 
+    function updateActive(active) {
+        setActive(active.padEnd(word.length));
```

Comment:
I feel like our dependence on needing to pad the strings is possibly a code smell (this goes true for how we're rendering the rows too).  Maybe we should just be storing the word, guesses, and evals all as arrays?  We seem to split the strings into arrays more than using them as plain strings.

Reply:
I think you're right yeah it would be better to make them arrays. This seems like it would be a more involved change can we find some time to go over this or maybe if you do one as an example I can finish the rest?

Reply:
This actually seems like it could be broken into 3 parts; 1 string padding, 2 row rendering, and 3 convert some strings to arrays. Is there a built in string to array function? Also do you mean storing guesses as like an array of arrays for each guess? I think evaluations is fine as is the values in it don't get split.

Reply:
Oh okay I see that would make sense then if thats the case then that should be easy.